.github/workflows/test.yml

```diff
@@ -37,18 +37,43 @@ jobs:
     name: Build-master
     runs-on: ubuntu-latest
     steps:
+    # Create a cache for the built master image
+    - name: Restore master image cache
+      id: image_cache_master
+      uses: actions/cache@v2
+      with:
+        path: |
+          /tmp/image_cache/
+        key: ${{ github.run_id }}-image-cache-master
+
+    - name: Check if master image build is needed
+      id: is_master_image_build_needed
+      continue-on-error: true
+      run: |
+        set -x
+        if [ -f /tmp/image_cache/image-master.tar.gz ]; then
+            mkdir -p dist/images/_output
+            cp /tmp/image_cache/image-master.tar.gz dist/images/_output/image-master.tar.gz
+            gunzip dist/images/_output/image-master.tar.gz
+            echo "::set-output name=MASTER_IMAGE_RESTORED::true"
```

Comment:
I'm guessing this is for setting your var to know if you restored from cache. How did you find this method of setting it like this?

Reply:
oh now I see some examples in: https://docs.github.com/en/actions/learn-github-actions/contexts#steps-context